tpp-backend/desktop-image-experiment/desktop.md

```diff
@@ -0,0 +1,68 @@
+## Creating a Hyper-V image
+
+Tried to do script this, failed, so here is the manual procedure to
+create, configure and possibly export a Hyper-V image for an OpenSAFELY
+backend.
+
+## Create Initial VM
+
+Use 'Quick Create' to launch 20.04 image from the Hyper-V Manager. This
+will download a customised 20.04 *desktop* image, which plays nice as a
+guest with Hyper-V. Note attempts to use a default server image have met
+with crushing failure and despair.
+
+This default image does not come with ssh installed. You must log on
+graphically and set that up. Go through the initial sign in stuff,
+create an account with same name as your github user, and whatever
```

Comment:
Why github user? Will this work for anyone, do they have to be in a certain organisation? Or is this just as a convenience for grabbing public keys? What are the consequences of my private key being leaked on this image?  (my understanding is that this is a temporary way to provide SSH access, the user will be removed at the end of the process, and github is just for ssh-pubkey-getting convenience, in which case I think we should say so)

Reply:
There is no clean up of the user. Hence why we suggest your gh account name, which will be full provisioned later by the scripts.

I think the desktop-image-experiment (which we are *not* using, as per my comment) has introduced some confusion. I will delete it.